public/javascripts/API.js

```diff
@@ -5,7 +5,7 @@
 var api = {
   login : function(params){
     $.ajax({
-      url : "http://" + window.location.host + "/api/auth",
+      url : "http://" + window.location.host + "/api/v1/login",
```

Comment:
I believe I put that there for people testing with docker's IP vs localhost
vs deployed

On Friday, May 27, 2016, Tim Richards <notifications@github.com> wrote:

> In public/javascripts/API.js
> <https://github.com/ripples/lv-client/pull/1#discussion_r64983068>:
>
> > @@ -5,7 +5,7 @@
> >  var api = {
> >    login : function(params){
> >      $.ajax({
> > -      url : "http://" + window.location.host + "/api/auth",
> > +      url : "http://" + window.location.host + "/api/v1/login",
>
> We should remove the http:// and window.location.host part.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ripples/lv-client/pull/1/files/1b7a51bbad066886623929d2b8169b59c3f14db4#r64983068>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AITAxF0EPnengn1xsKOBdKc3Q6LSnIzOks5qF7EPgaJpZM4IljfX>
> .
>